json-rpc/src/tests/mock_db.rs

```diff
@@ -104,7 +116,22 @@ impl LibraDBTrait for MockLibraDB {
         let first_transaction_version = transactions.first().map(|_| start_version);
         Ok(TransactionListWithProof {
             transactions,
-            events: None,
+            events: if fetch_events {
+                Some(
+                    (start_version..start_version + limit)
+                        .map(|version| {
+                            self.events
+                                .iter()
+                                .filter(|(v, _)| *v == version)
```

Comment:
nit, can ignore: we can also use `filter_map` :D

Reply:
kina of awkward to have to write if ... { Some() } else {None }